spec/lib/whitehall_importer/integrity_checker/change_history_check_spec.rb

```diff
@@ -0,0 +1,138 @@
+RSpec.describe WhitehallImporter::IntegrityChecker::ChangeHistoryCheck do
+  let(:published_edition) do
+    build(:edition,
+          document_type_id: "news_story",
+          state: "published")
+  end
+  let(:draft_edition) { build(:edition, document_type_id: "news_story") }
+  let(:public_timestamp) { Time.zone.now.yesterday }
+
+  describe "#match?" do
+    it "returns true if the proposed change history matches the Publishing API history for a non-draft edition" do
+      proposed_change_history = [
+        {
+          note: "First published.",
+          public_timestamp: public_timestamp,
+        },
```

Comment:
We might find we can make this test a bit shorter by having a basic method of:

```
def change_note(note, timestamp) 
  { note: note, public_timestamp: timestamp.rfc3339 }
end
```

that we can use each time we need a change note for this